test/verify/machineslib.py

```diff
@@ -1605,6 +1605,13 @@ def getSourceTypeLabel(sourceType):
                 else:
                     b.select_from_dropdown("#storage-size-unit-select", self.storage_size_unit)
                     b.set_input_text("#storage-size", str(self.storage_size))
+                    # helpblock will be missing if available storage size could not be calculated (no default storage pool found)
```

Comment:
I'm trying to find default pool in our tests but it seems there ou tests do not always have default pool present.

Reply:
Please find out which libvirt package creates the default storage pool , I think it's libvirt-daemon-driver-storage.x86_64 but please verify.

In case it's missing from some distros so no default storage pools is created, just create the /var/lib/libvirt/images directory which will be used.